sampler_t.adoc

```diff
@@ -46,7 +46,7 @@ The sampler fields are described in the table below:
 
 |
 
-Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized or unnormalized values.
+Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized ([0,1[) or unnormalized ({0, size-1}) values.
```

Comment:
Thanks, I forgot the points.

Actually, I don't know if `{}` also implies integer coordinates, `{0.0, 1.0, 2.0, ...}` would also be possible, I guess. Anyway, I think it is clear know.